Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:jammy
+
+RUN apt update
+RUN DEBIAN_FRONTEND=noninteractive apt install -y python3-pip ansible wget git
+RUN pip3 install purefusion
+
+RUN mkdir /workdir
+WORKDIR /workdir
+
+RUN wget -q --show-progress -O /usr/bin/hmctl https://github.com/PureStorage-OpenConnect/hmctl/releases/latest/download/hmctl-linux-amd64
+CMD chmod +x /usr/bin/hmctl
+
+RUN ansible-galaxy collection install purestorage.fusion
+
+RUN mkdir -p /root/.pure
+ENV PRIV_KEY_FILE=/workdir/lab_key.pem
+RUN echo  "{ \"default_profile\": \"pm-lab-admin\",\n" \
```

Comment:
Let's call this profile `main` instead.

And call `lab_key.pem` `private_key.pem`.

This is more general.